feature/bgp/policybase/otg_tests/nested_policies/README.md

```diff
@@ -0,0 +1,293 @@
+    *   /routing-policy/policy-definitions/policy-definition/statements/statement/actions/config/policy-result
+##### Configure BGP actions to set local-pref
+*   For routing-policy ```lp-policy-v4``` statement ```lp-statement-v4``` set local-preference to ```200```
+    *   /routing-policy/policy-definitions/policy-definition/statements/statement/actions/bgp-actions/config/set-local-pref
+
+##### Configure a nested policy 
+*   For routing-policy ```lp-policy-v4``` attach the policy ```match-policy-v4```
+    *   /routing-policy/policy-definitions/policy-definition/statements/statement/conditions/config/call-policy
+
+##### Configure the parent bgp import policy for the DUT BGP neighbor on ATE Port-1
+*   Set default import policy to ```REJECT_ROUTE```
+    *   /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/config/default-import-policy
+*   Apply the parent policy ```lp-policy-v4``` to the BGP neighbor
+    *   /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/config/import-policy
+
+##### Verification
+*   Verify that the parent ```lp-policy-v4``` policy is successfully applied to the DUT BGP neighbor on ATE Port-1
+    *   /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/state/import-policy
+*   Verify that the parent ```lp-policy-v4``` policy has a child policy ```match-policy-v4``` attached
+    *   /routing-policy/policy-definitions/policy-definition/statements/statement/conditions/state/call-policy
+
+##### Validate test results
+*   Validate that the DUT receives the prefix ```ipv4-network-1``` i.e. ```192.168.10.0/24``` from BGP neighbor on ATE Port-1
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv4-unicast/loc-rib/routes/route/prefix
+*   Validate that the prefix ```ipv4-network-1``` i.e. ```192.168.10.0/24``` from BGP neighbor on ATE Port-1 has local preference set to 200
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/attr-sets/attr-set/state/med
+*   Initiate traffic from ATE Port-2 towards the DUT destined to ```ipv4-network-1``` i.e. ```192.168.10.0/24```
+    *   Validate that the traffic is received on ATE Port-1
+
+### RT-1.30.2 [TODO: https://github.com/openconfig/featureprofiles/issues/2608]
+#### IPv4 BGP nested export policy test
+---
+##### Configure a route-policy to prepend AS-PATH
+*   Configure an IPv4 route-policy definition with the name ```asp-policy-v4```
+    *   /routing-policy/policy-definitions/policy-definition/config/name
+*   For routing-policy ```asp-policy-v4``` configure a statement with the name ```asp-statement-v4```
+    *   /routing-policy/policy-definitions/policy-definition/statements/statement/config/name
+*   For routing-policy ```asp-policy-v4``` statement ```asp-statement-v4``` set policy-result as ```ACCEPT_ROUTE```
```

Comment:
Adding ACCEPT_ROUTE as a statement a the parent policy-definition before the `call-policy` will cause the the parent policy `asp-policy-v4` stop evaluation and not proceed to `med-policy-v4` [per specification](https://github.com/openconfig/public/blob/master/release/models/policy/openconfig-routing-policy.yang#L51-L58))

For example, this will read as:
```
asp-policy-v4
  as path prepend and accept route. 
  call policy med-policy-v4   
```

To test nested policy we need some scenario where matched routes don't have a have ACCEPT or REJECT `policy-result`.  (ie: they don't match the or NEXT_STATEMENT is used) in the first policy.  Then the second policy is executed where some action happens (like setting med which you do have).

```
asp-policy-v4
  as path prepend and next statement
  call policy med-policy-v4   
```

```
med-policy-v4
   set med 1000 and accept route
````